src/app.ts

```diff
@@ -303,6 +304,39 @@ export const start = async (args: string[]) => {
     }
   });
 
+  branchTable.key("d", function () {
+    const selection = parseSelection(
+      branchTable.items[branchTable.selected].content,
+    );
+    const branchName = selection[2];
+
+    if (state.currentRemoteIndex !== 0) {
+      logger.error("Branch deletion is only available on heads");
+      return;
+    }
```

Comment:
Bah there's a bug here as heads might not be index 0 always. So IF we decide to go ahead with this approach, I'll fix this up